_posts/2021-01-12-december-rubygems-update.md

```diff
@@ -0,0 +1,46 @@
+---
+title: December 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in December.
+
+## RubyGems News
+In December, we released `bundler 2.2` and `rubygems 3.2` 🎉. We implemented a unified release workflow and fixed a longstanding gem install issue. - [#4110](https://github.com/rubygems/rubygems/pull/4110)
+
+We killed a couple of regressions in `bundler 2.2.0` and `rubygems 3.2.0` and released a couple of patch level versions for each of them. 
+
+Finally, we fixed a gem command that was crashing or asking for OTP twice when it prompted for an email and password. - [#4162](https://github.com/rubygems/rubygems/pull/4162)
+
+This month, [RubyGems](https://github.com/rubygems/rubygems) gained [203 new commits](https://github.com/rubygems/rubygems.org/compare/master@%7B2020-12-01%7D...master@%7B2020-12-21%7D), contributed by 13 authors. There were 4191 additions and 2066 deletions across 1184 files.
+
+## RubyGems.org News
+This month, we published a guide on RubyGems.org about [API keys, their scopes, and CLI usage](https://guides.rubygems.org/api-key-scopes/) - [#275](https://github.com/rubygems/guides/pull/275), we investigated and removed `ruby-bitcoin` and `pretty_color` gems for containing ransomware, this issue was reported by [@mensfeld](https://github.com/mensfeld) for obfuscated code. - [Gems yanked and Accounts Locked Wiki](https://github.com/rubygems/rubygems.org/wiki/Gems-yanked-and-accounts-locked#14-dec-2020)
```

Comment:
> gems for containing ransomware

I know, I must have said it was ransomware in my updates, however, it was not ransomware. let's update it to "containing malware" or "containing malicious code which could steal sensitive information".

Reply:
Alright. I'll add the update.